templates/_helpers.yaml

```diff
@@ -79,11 +79,11 @@
 {{- end }}
 
 {{ define "airflow_url" -}}
-{{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
+app.local.astronomer-development.com
```

Comment:
This change is wrong for non-dev I think

Reply:
These could both be `printf "app.%s" .Values.global.baseDomain`